Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Stage 1: Build the application
+FROM maven:3.9.6-amazoncorretto-21 as builder
+
+# Set the working directory in the Maven image
+WORKDIR /app
+
+# Copy the java source files and the pom.xml file into the image
+COPY src ./src
+COPY pom.xml .
+
+# Build the application
+RUN mvn clean package
```

Comment:
I would -DskipTests

Reply:
Why? Keep in mind that we leave out integration tests with this command and run only unit tests when https://github.com/cBioPortal/cbioportal-core/pull/29 is in.